Bottom: b1a52c2ea845b8325718af1260e8296f1c6097d7
Top:    4c39bbe9d9967e123bc70f80985ca589bb10cdd6
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-05-25 10:35:49 -0500

Refresh of all-normalize-vars

---
diff --git a/roles/docker-exim4/tasks/main.yml b/roles/docker-exim4/tasks/main.yml
index 5900753..4594a26 100644
--- a/roles/docker-exim4/tasks/main.yml
+++ b/roles/docker-exim4/tasks/main.yml
@@ -10,7 +10,7 @@
 - name: restore dir
   file:
     state: directory
-    path: '{{ host_exim4_docker_restore_dir }}'
+    path: '{{ exim4_docker_restore_dir }}'
     owner: root
     group: tape
     mode: 'u=rwx,g=rwx,o=rx'
diff --git a/roles/docker-gitlab/templates/config.sh.j2 b/roles/docker-gitlab/templates/config.sh.j2
index e2dd809..4df5fa3 100644
--- a/roles/docker-gitlab/templates/config.sh.j2
+++ b/roles/docker-gitlab/templates/config.sh.j2
@@ -10,7 +10,7 @@ DOCKER_IMAGE_TAG='{{ docker_image_tag }}'
   GITLAB_DB_CONTAINER_NAME='{{ gitlab_db_container_name }}'
   GITLAB_REDIS_CONTAINER_NAME='{{ gitlab_redis_container_name }}'
   GITLAB_DV_NAME='{{ gitlab_dv_name }}'
-  HOST_GITLAB_RESTORE_DIR='{{ host_gitlab_docker_restore_dir }}'
+  HOST_GITLAB_RESTORE_DIR='{{ gitlab_docker_restore_dir }}'
   HOST_GITLAB_BACKUP_DIR='{{ gitlab_docker_backup_dir }}'
 
   # this file should be only readable by root since username and password exist in the file in plain text
diff --git a/roles/docker-mantisbt/templates/config.sh.j2 b/roles/docker-mantisbt/templates/config.sh.j2
index 7e5bbc5..f9538c8 100644
--- a/roles/docker-mantisbt/templates/config.sh.j2
+++ b/roles/docker-mantisbt/templates/config.sh.j2
@@ -6,7 +6,7 @@
   MANTISBT_DB_CONTAINER_NAME='{{ mantisbt_db_container_name }}'
   MANTISBT_DV_NAME='{{ mantisbt_dv_name }}'
   MANTISBT_DB_DV_NAME='{{ mantisbt_db_dv_name }}'
-  HOST_MANTISBT_RESTORE_DIR='{{ host_mantisbt_docker_restore_dir }}'
+  HOST_MANTISBT_RESTORE_DIR='{{ mantisbt_docker_restore_dir }}'
   HOST_MANTISBT_BACKUP_DIR='{{ mantisbt_docker_backup_dir }}'
   STATIC_BACKUP_FILE='{{ mantisbt_static_backup_file }}'
   DATABASE_BACKUP_FILE='{{ mantisbt_database_backup_file }}'
diff --git a/roles/docker-mediawiki/templates/config.sh.j2 b/roles/docker-mediawiki/templates/config.sh.j2
index db89e99..28f6a40 100644
--- a/roles/docker-mediawiki/templates/config.sh.j2
+++ b/roles/docker-mediawiki/templates/config.sh.j2
@@ -6,7 +6,7 @@
   WIKI_DB_CONTAINER_NAME='{{ wiki_db_container_name }}'
   WIKI_DV_NAME='{{ wiki_dv_name }}'
   WIKI_DB_DV_NAME='{{ wiki_db_dv_name }}'
-  HOST_WIKI_RESTORE_DIR='{{ host_wiki_docker_restore_dir }}'
+  HOST_WIKI_RESTORE_DIR='{{ wiki_docker_restore_dir }}'
   HOST_WIKI_BACKUP_DIR='{{ wiki_docker_backup_dir }}'
   STATIC_BACKUP_FILE='{{ wiki_static_backup_file }}'
   DATABASE_BACKUP_FILE='{{ wiki_database_backup_file }}'
